new/LIT7376MGBone.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7376MGBone" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: St George finds a bone belonging to a Muslim whose soul can find no rest; through the intercession of the saint, the man is resuscitated to convert and to be forgiven his sins</title>
```

Comment:
'resuscitated to be forgiven his sins' sounds strange for me, maybe is resuscitated to convert and to confess his sins?